packages/claude-hooks/src/base-hook.ts

```
@@ -0,0 +1,166 @@
+        this.log.debug(
+          `Parsed ${this.eventType} event: ${JSON.stringify(event)}`,
+        )
+        return event
+      } else {
+        this.log.warning(
+          `Expected ${this.eventType} event, got type=${event.type}, hook_event_name=${event.hook_event_name}`,
+        )
+        return null
+      }
+
+      return event
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)